app/customer.py

```diff
@@ -0,0 +1,20 @@
+from app.car import Car
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: [int],

```

Comment:
For the 'location' parameter, use Tuple[int, int] instead of [int] to represent coordinates (x, y).